services/AVS/privateClouds/templates/policy-arm/CPUCritical_bb83ae53-3b91-49ff-a351-9545ea747335.json

```diff
@@ -0,0 +1,388 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-08-01/managementGroupDeploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "displayName": {
+      "defaultValue": "Deploy AVS privateClouds EffectiveCpuAverage Alert",
+      "type": "String",
+      "metadata": {
+        "description": "Display name of policy"
+      }
+    },
+    "description": {
+      "defaultValue": "Policy to Audit/Deploy AVS privateClouds EffectiveCpuAverage Alert",
+      "type": "String",
+      "metadata": {
+        "description": "Description of policy"
+      }
+    },
+    "currentDateTimeUtcNow": {
+      "defaultValue": "[utcNow()]",
+      "type": "String",
+      "metadata": {
+        "description": "The current date and time using the utcNow function. Used for deployment name uniqueness"
+      }
+    },
+    "telemetryOptOut": {
+      "defaultValue": "No",
+      "allowedValues": [
+        "Yes",
+        "No"
+      ],
+      "type": "String",
+      "metadata": {
+        "description": "The customer usage identifier used for telemetry purposes. The default value of False enables telemetry. The value of True disables telemetry."
+      }
+    }
+  },
+  "variables": {
+    "pidDeploymentName": "[take(concat('pid-8bb7cf8a-bcf7-4264-abcb-703ace2fc84d-', uniqueString(managementGroup().id, parameters('currentDateTimeUtcNow'))), 64)]",
```

Comment:
I just tried to deploy this specific file and it deployed ok for me.  Can you tell me how you are trying to deploy it?  I just did a Custom Deployment in the portal, loaded the file and deployed without error.

Reply:
@arjenhuitema I looked at this again and I think it may be an issue with the template validator in VS Code.  As far as I can tell 'managementGoup' **is** a valid template function.  See here https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-scope#managementgroup

Reply:
Loaded template file shows no error on line 39 when loaded into portal:
<img width="862" alt="image" src="https://github.com/user-attachments/assets/2574c1cd-6587-4f1c-8b24-9a2136a5a9ad" />

Reply:
Appears to be known issue:  https://github.com/microsoft/vscode-azurearmtools/issues/1428